addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -170,4 +170,24 @@
 		<state min="0" max="360" step="1" readOnly="true"></state>
 	</channel-type>
 
+	<channel-type id="scene">
+		<item-type>Number</item-type>
+		<label>Scene</label>
+		<description>Scene channel</description>
+		<state readOnly="false">
+			<options>
+				<option value="0">Scene 1</option>
```

Comment:
b. Yes, your understanding is correct. Although, the same behavior could be achieved using rules, I've intentionally implemented scene switch abstraction layer for the following reasons:
1) Existing Command channel doesn't provide reliable on/off functionality as mentioned above and Scene Switch is intended to address that limitation.
2) It will be pretty straightforward for the users to "reconnect" switches on their dashboards to the different channels (Scene Switch instead of Command) to start using new reliable ON/OFF functionality rather than creating extra rules. It's just a matter of few clicks in PaperUI. We need to keep in mind that as IoT is getting to the masses, there will be more and more users with limited skills in scripting/development that prefer GUI way of doing things. And that's one of the goals of openHAB 2 from my understanding.